hnn_core/viz.py

```diff
@@ -7,7 +7,21 @@
 from itertools import cycle
 
 
-def _get_plot_data(dpl, layer, tmin, tmax):
+def _check_scaling_units(scaling, units):
+    if not isinstance(scaling, float):
+        if scaling is None:
+            scaling = 1  # allow implicitly asking for no scaling
+        else:
+            raise ValueError(f'scaling must be a float, got {type(scaling)}')
+    if not isinstance(units, str):
+        if units is None:
+            units = ''  # allow implicitly asking for no scaling
+        else:
+            raise ValueError(f'units must be a string, got {type(units)}')
+    return scaling, units
```

Comment:
```suggestion
```

I don't think we need this level of complexity. The less code the better. You'll find me complaining if there are too many green lines that are not documentation ;-)

Reply:
The reasons for the private function were:

1. scaling is applied in `plot_dipole` and `plot_psd`, so to avoid duplicate code
2. dealing with scaling is still undecided `hnn-core`-wide: *what should the sane defaults be*?